pkg/entities/entities_api.go

```diff
@@ -1320,28 +1592,6 @@ const getEntityQuery = `query(
 		}
 		agentInstallType
 		applicationId
-		browserSettings {
-			browserConfig {
-				apdexTarget
-			}
-			browserMonitoring {
-				ajax {
-					denyList
-				}
-				loader
-				distributedTracing {
-					allowedOrigins
-					corsEnabled
-					corsUseNewrelicHeader
-					corsUseTracecontextHeaders
-					enabled
-					excludeNewrelicHeader
-				}
-				privacy {
-					cookiesEnabled
-				}
-			}
-		}
```

Comment:
@sanderblue looks like Tutone's removing this portion, since the deepest (nested) fields of those at depth:2 are at a depth greater than 2. I just ran a `tutone generate --refetch --package entities` with 

```yml
          - name: entity
            max_query_field_depth: 3
            # changed 2 to 3 above
```

and this brought back this attribute `browserSettings`, with all of its nested attributes. While this might be what we'd want to do, this comes with the slight downside of modifying the entire query with additional fields, since the depth is now 3. 

What do you think we should do now - given that this block (unfortunately) is necessary and cannot be discarded (this is used by the Terraform resource `newrelic_browser_application`); do you think a change can be made with the query regenerated with depth 3? A point to also note is that the regeneration of this query with depth 3 doesn't seem to add any changes to `entities/types.go`, given it already seems to have types with their nested contents defined with the highest depth.

Reply:
Interesting, I'm surprised this didn't break tests, but you're right we can't remove this. I'll update with your suggestion and see how things go.